ReText/tab.py

```diff
@@ -45,12 +51,14 @@
 	ReTextWebEnginePreview = None
 
 PreviewDisabled, PreviewLive, PreviewNormal = range(3)
+HTML_IMG_TAG_SRC_RE = re.compile(r'src[\s]*=[\"\']([^\"\'>]+)[\"\']\/?>')
```

Comment:
This will match not only `<img>` tags, but also other tags with `src` attribute, such as `<script>`.

Reply:
You're right, my mistake. Fixed.